### PR TITLE
feat: fail early when trying to send empty payload

### PR DESCRIPTION
**Pull request by fryorcraken**

### CODE REVIEW

**Comment by fryorcraken** [comment]
@fbarbu15 limitation of using js-waku CI is that js-waku does a number of check to avoid network access when it already know it will fail.

Which means that to properly test a remote node (e.g. nwaku) then we may need to add (separate) tests that skip this kind of checks.

**Comment by fbarbu15** [comment]
> @fbarbu15 limitation of using js-waku CI is that js-waku does a number of check to avoid network access when it already know it will fail.
> 
> Which means that to properly test a remote node (e.g. nwaku) then we may need to add (separate) tests that skip this kind of checks.

I see, maybe those types of scenarios can be better covered:
- On nwaku/gowaku side by Alex/Roman
- During interop testing

File: packages/core/src/lib/light_push/index.ts [hunk 1]
```diff
@@ -56,7 +56,12 @@ class LightPush extends BaseProtocol implements ILightPush {
     pubSubTopic: string
   ): Promise<PreparePushMessageResult> {
     try {
-      if (!isSizeValid(message.payload)) {
+      if (!message.payload || message.payload.length === 0) {
```
Comment (weboko) [inline packages/core/src/lib/light_push/index.ts:59]: I guess if we continue extending validation for `payload` then maybe makes sense to have `validatePayload` function?

Comment (fryorcraken) [reply packages/core/src/lib/light_push/index.ts:59]: Yes I considered it and went for this small PR to fix the issue and then started https://github.com/waku-org/js-waku/pull/1643 to fix wider issue around validation.

**Pull request merged**